input/pagecontent/StructureDefinition-au-erequesting-imaging-request-intro.md

```diff
@@ -0,0 +1,9 @@
+### Usage scenarios
+The following are supported usage scenarios for this profile:
+
+- Query for a request for a diagnostic imaging service for a patient
```

Comment:
The usage scenario wording reads like AU Core. Remove for now, this should be added after the interactions and obligations meetings.

Reply:
Removed

Reply:
Remove it entirely.

Reply:
Done